charts/rancher-webhook/rancher-webhook/0.1.0/Chart.yaml

```diff
@@ -8,4 +8,4 @@ apiVersion: v2
 appVersion: 0.1.0
 description: ValidatingAdmissionWebhook for Rancher types
 name: rancher-webhook
-version: 0.1.0+up0.1.0
+version: 0.1.0
```

Comment:
The `+up` part is omitted if the rancher chart and the upstream chart have the same version.